docs/podman-restart.1.md

```diff
@@ -12,33 +12,51 @@ Containers will be stopped if they are running and then restarted. Stopped
 containers will not be stopped and will only be started.
 
 ## OPTIONS
-**--timeout**
-
-Timeout to wait before forcibly stopping the container
+**--all, -a**
+Restart all containers regardless of their current state.
 
 **--latest, -l**
-
 Instead of providing the container name or ID, use the last created container. If you use methods other than Podman
 to run containers such as CRI-O, the last started container could be from either of those methods.
 
+**--running-only**
```

Comment:
This is just  "--running" now me thinks.

Reply:
fixed